packages/web-fragments/src/elements/reframed/reframed.ts

```diff
@@ -4,6 +4,7 @@ type ReframedOptions = {
 	bound: boolean;
 	container: HTMLElement;
 	headers?: HeadersInit;
+	name: string;
```

Comment:
I was wondering if we should prefix all names and ids with "wf-" by default, to unexpected collisions in the app shell

Reply:
maybe? let's discuss. I'll merge this as is an we can still change it.